CMakeLists.txt

```diff
@@ -498,7 +499,7 @@ ADD_CUSTOM_TARGET(dist
 			COMMAND make clean
 			COMMAND rm -rf "${TMP}"
 			COMMAND mkdir -p "${TMP}"
-			COMMAND cp AUTHORS build_mingw32 build_mingw64 CMakeLists.txt configure COPYING INSTALL lmms.1 lmms.rc.in lmms.spec.in lmmsconfig.h.in lmmsversion.h.in README TODO "${TMP}"
+			COMMAND cp AUTHORS build_mingw32 build_mingw64 CMakeLists.txt configure COPYING INSTALL README.md TODO "${TMP}"
```

Comment:
Also, should TODO and configure be removed, as well as build_mingw32/64 since they were moved to a different directory just like the lmms.* files?